resources/popup.html

```diff
@@ -39,7 +40,7 @@
                 </a>
             </div>
             <div class="col s6">
-                <a class="btn waves-effect lime waves-light tooltipped" href="../analytics/index.html" target="_blank"
+                <a id="analytics-link" class="btn waves-effect lime waves-light tooltipped" target="_blank"
```

Comment:
Because of extensibility. Now a page only needs a meta-data element to determine the target. I did remove the href attribute though, thanks for noticing